bin/compile.js

```diff
@@ -79,7 +79,7 @@ async function compile(sourceDir, targetDir, config) {
     const component = createComponent({
       html: fs.readFileSync(f, 'utf8'),
       name: filename,
-      ...config
+      ...Object.assign({}, config, { version })
```

Comment:
Well seen!
It may be more straightforward to write
```
const component = createComponent({
      html: fs.readFileSync(f, 'utf8'),
      name: filename,
      version,
      ...Object.assign({}, config),
  })
```

Reply:
I think that is more readable! If you would like to make the change, feel free to do so and close this PR. Alternatively I can make them qhen I get home from work, either's good for me.

Reply:
I'd rather have you to update it just to ensure you remain the author.
Any time you get the chance to do it, no rush.